site/docs/components/index.mdx

```diff
@@ -20,7 +20,7 @@ with designs for light and dark modes in all four densities.
 
 Stable components do not feature an accompanying symbol in their navigation link titles. You can find them in the `@salt-ds/core` package.
 
-The {meta.labsIcon} symbol denotes components that are under active development. These may not function as expected and may change due to feedback. You should exercise caution. You can find them in the `@salt-ds/lab` package.
+The {meta.labsIcon} symbol denotes components that are under active development. These may not function as expected and may change due to feedback. You should exercise caution. You can find them in the `@salt-ds/lab` package. The use of lab components not listed on the website is not encouraged since they aren't receiving active development.
```

Comment:
```suggestion
The {meta.labsIcon} symbol denotes components that are under active development. These may not function as expected and may change due to feedback. You should exercise caution. You can find them in the `@salt-ds/lab` package.

**Note:** The use of lab components not listed on the website isn't encouraged since they aren't receiving active development.
```

thought's is not -> isn't is following our guidance
and should be make this stand out more?